src/services/permit2/utils/config.ts

```diff
@@ -11,15 +14,15 @@ export const PERMIT2_SUPPORTED_CHAINS = [
   Chains.FANTOM,
   Chains.ARBITRUM,
   Chains.OPTIMISM,
-  Chains.BASE_GOERLI,
+  Chains.BASE,
   Chains.MOONRIVER,
   Chains.MOONBEAM,
   Chains.FUSE,
   Chains.EVMOS,
   Chains.CELO,
   Chains.GNOSIS,
   Chains.KAVA,
-  Chains.POLYGON_ZKEVM,
```

Comment:
Actually, instead of removing this, we could continue to use the previous version

Reply:
Nice feature!